core/lib/mno_enterprise/concerns/models/ability.rb

```diff
@@ -141,7 +141,7 @@ def impac_abilities(user)
 
   # Abilities for admin user
   def admin_abilities(user)
-    if user.admin_role == 'admin'
+    if user.admin_role.try(:downcase) == 'admin'
```

Comment:
Updated, let me know if good and I'll squash the commits once the specs are green